NEWS.md

```diff
@@ -1,5 +1,16 @@
 Thank you to all the [contributors](https://github.com/thoughtbot/clearance/graphs/contributors)!
 
+New for 1.8.0 (January 23, 2015)
+* Fixed an issue that would cause sites still using the deprecated `authorize`
```

Comment:
"site **that** still using"?

Feel free to leave it if that's already grammatically correct.